sssom/cliques.py

```diff
@@ -95,7 +108,13 @@ def invert_dict(d: Dict[str, str]) -> Dict[str, str]:
     return invdict
 
 
-def get_src(src, cid):
+def get_src(src: str, cid: str):
```

Comment:
src should be Optional[str] -> see the None check in the definition of the function